cli/server/spreadsheetRoutes.ts

```diff
@@ -1012,6 +1186,143 @@ function exportAsJSON(controls: any[], metadata: any, res: express.Response) {
 	res.json(exportData);
 }
 
+// Get available column headers for export
+router.get('/:sheetDir/export-column-headers', async (req, res) => {
+	try {
```

Comment:
Trying to track down the need for this "sheet specific" route and I'm wondering if its really necessary? Looking at `export-controls` route, we are able to pull the lula.yaml without a specific sheet directory - https://github.com/defenseunicorns/lula/blob/main/cli/server/spreadsheetRoutes.ts#L716-L738. If I'm following this correctly `state.CONTROL_SET_DIR` should be the actual control set (sheet) directory already?

Reply:
I was missing `getCurrentControlSetPath()` when I was looking for the file which set it from the base to the spreadsheet dir. This should be all resolved now